e2e-tests/popup/account/remove-imported-account.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  ACCOUNT_NAMES,
+  IMPORTED_PEM_ACCOUNT,
+  secretKeyPathForPEM
+} from '../../constants';
+import { popup, popupExpect } from '../../fixtures';
+
+popup.describe('Popup UI: remove imported account', () => {
+  popup(
+    'should import account via .pem file',
```

Comment:
we already have a test for importing accounts via files

Reply:
name is wrong, test is about removing imported account. Corrected